### DISCUSSION TITLE
Streaming to a server

### DISCUSSION THREAD

**Discussion by danbachar** [rzeldent/esp32cam-rtsp#149] [General]
Hey all!

First of all, thanks a lot for writing this library for the ESP32. On my search for a viable streaming solution for my esp32cam I stumbled across this project, and I hope it could be a good match for my use case. I hope I understood this protocol correctly, but probably this is not the right place to ask questions about it.

Anyway, I would like to use the ESP32-CAM to constantly stream images to my raspberry pi - the pi acts as the receiving server, and the cam uploads its data there. I'm pretty sure your use case is the opposite, where the camera acts as a web server as well.
Do you think I could use this library for my use case?

Thanks,
Dan

**Comment by rzeldent** [comment]
Hi Dan,

I think it is possible. Just install VLC on the PI and connect to the ESP32 CAM.
For persistent connections this might be not the right solution; WiFi is not always stable.

Give it a try and good luck with your project!
